scripts/deploy.ts

```diff
@@ -102,16 +104,21 @@ function main() {
   makeCommonPost(buildObj)
 
   // buildCommonPre()
-  if (argv[3] === 'ios') {
-    buildIos(buildObj)
-  } else if (argv[3] === 'android') {
+  if (buildObj.platformType === 'ios') {
+    if (buildObj.simBuild) {
+      buildIosSim(buildObj)
+    } else {
+      buildIos(buildObj)
+    }
+  } else if (buildObj.platformType === 'android') {
```

Comment:
I was initially expecting more of the sim vs prod ios builds to share code. In the end they are pretty different paths so it's fair to move it to `ios-sim`

Reply:
Actually, after trying this change, I decided to only allow `ios-sim` as a command line arg. The remaining code will keep `simBuild` separate from `platformType` since `platformType` is assumed in several places to be `ios` or `android` and is even used to creates necessary paths.